### ISSUE TITLE
SMN request: Gemshine to Ruin III and Precious Brilliance to Outburst

### ISSUE THREAD

**Issue by evnhali** [daemitus/XIVComboPlugin#23]
Could we please have an option to change Gemshine to Ruin/Ruin III/Astral Impulse/Fountain of Fire and Precious Brilliance to Outburst/Tri-Disaster/Astral Flare/Brand of Purgatory when not attuned? 

The Enkindle feature seems really awkward (replacing two GCD filler spells with one oGCD ability on a 20s CD?) and does nothing below level 70, while the Ruin III and Outburst buttons feel duplicative.

**Comment by daemitus** [comment]
are all of the gemshine variants on the gcd?

**Comment by evnhali** [comment]
Yes, unless something changes above level 70 that I haven't been able to figure out from the tooltips.

**Comment by daemitus** [comment]
So: 
 - if attuned: gemshine variant
 - if no attunement is up, and no baha/phoenix, and ruin4 available, ruin4. (you wouldn't want to use ruin4 during baha/phoenix right?)
 - otherwise ruin3
```
                if (level >= SMN.Levels.Gemshine && (gauge.IsIfritAttuned || gauge.IsTitanAttuned || gauge.IsGarudaAttuned))
                    return OriginalHook(SMN.Gemshine);

                if (level >= SMN.Levels.Ruin4 && gauge.AttunmentTimerRemaining == 0 && HasEffect(SMN.Buffs.FurtherRuin))
                    return SMN.Ruin4;
```

**Comment by daemitus** [comment]
well, otherwise ruin3 or whatever it changes to with baha or phoenix

**Comment by neongrey** [comment]
hmmm, yeah, I think that would work well-- i do best feeling these things out in practice but yeah, that sounds like a good way of handling it.

and yeah you dont want to use ruin 4 during baha/phoenix, you wanna be spamming those since those are your only strictly timed phases, everything else is 30 seconds to use 2-4 shots.

**Comment by evnhali** [comment]
That sounds right! Thank you!

**Comment by daemitus** [comment]
Ok, im offering two ways of playing with this most recent push.

You can use ruin + gemburst on two buttons. ruin > ruin4 and gemburst >
enkindle.
Alternatively, ruin > gemburst > ruin4, and keep enkindle and astral.

**Comment by evnhali** [comment]
Thanks! Was that change meant to be included with the updates you pushed on 12/8? My ingame XIVCP-E hasn't updated and when I tried manually downloading from your repo it gave me the same version I currently have.

**Comment by daemitus** [comment]
I forget  I need to do that sometimes. expect it soon.

**Comment by neongrey** [comment]
beautiful, got it, will give it a go and see how it feels!

**Comment by neongrey** [comment]
need to downscale to 50/60/70 but roulettes havent been playing nice, and i'm not terribly concerned just with how button flow works in smn; at 80/90 it feels _fantastic_, no real pain in my hand whatsoever anymore... we'll see how long _that_ lasts but i was feeling some strain before. thanks so much!!!

**Comment by Jinxit** [comment]
This doesn't seem to work for me at 90. Both Shiny Outburst and Further Shiny Outburst seem to not do anything, whereas Shiny Ruin works exactly as expected.

**Comment by neongrey** [comment]
god you're right on the gemburst/shiny gemburst, i thought i was misunderstanding something as i was rearranging my hotbar but you're right it's not doing the thing

**Comment by daemitus** [comment]
ok, the not changing thing is fixed. i need to look through the rest of what needs doing

**Comment by daemitus** [comment]
Open up a new issue if this needs tweaking again. Please look at what's available in `1.3.0.23`
